src/renderers/webgl/WebGLProgram.js

```diff
@@ -540,9 +540,9 @@ function WebGLProgram( renderer, extensions, code, material, shader, parameters
 
 	gl.linkProgram( program );
 
-	var programLog = gl.getProgramInfoLog( program );
-	var vertexLog = gl.getShaderInfoLog( glVertexShader );
-	var fragmentLog = gl.getShaderInfoLog( glFragmentShader );
+	var programLog = gl.getProgramInfoLog( program ).replace(/\0/g, '').trim();
```

Comment:
This isn't going to fix this issue for all cases.

As the developers at chromium stated, the information here is driver implementation specific. In my case, this returns 4 blank lines on some occasions.

Also maybe it's not a good idea to use regex in this part of the code for performance reasons.